import_ldraw.py

```diff
@@ -928,6 +964,41 @@ def getColorValue(line, value):
     if value in line:
         n = line.index(value)
         return line[n + 1]
+        
+def linkedParts():
+    """Clean-up design by linking identical parts (mesh/color)"""
```

Comment:
Docstrings always end in a period and, if applicable, annotate the parameters/return values. Check out #101 for examples on how it's done. :)